eleventy.config.js

```diff
@@ -1,9 +1,15 @@
+const eleventyAutoCacheBuster = require("eleventy-auto-cache-buster");
```

Comment:
https://cassey.dev/11ty-on-cloudflare-pages/ cloudflare caches aggressively and this was confusing, but this plugin will fix it. This line & line 4 are the only changes needed for it.

Reply:
ok and the package.json addition.